gateleen-hook/src/main/java/org/swisspush/gateleen/hook/reducedpropagation/ReducedPropagationManager.java

```diff
@@ -173,7 +173,7 @@ private void processExpiredQueues(String lockToken) {
                         if (!OK.equals(event1.result().body().getString(STATUS))) {
                             log.error("Failed to process expired queue '"+expiredQueue+"'. Message: " + event1.result().body().getString(MESSAGE));
                         } else {
-                            log.info(event1.result().body().getString(MESSAGE));
+                            log.debug(event1.result().body().getString(MESSAGE));
```

Comment:
(*minor*) I think here it would make sense to use a guard:
`if(log.isDebugEnabled()){ ... }`
this would prevent from calling `getString` everytime. Which *might* be an expensive operation (I did not test it)